packages/website/.vitepress/theme/custom.css

```
@@ -0,0 +1,245 @@
+/* ── Breathing Friction Node ✦ ──────────────────────────── */
+.friction-node {
+  width: 9px;
+  height: 9px;
+  border-radius: 999px;
+  background: var(--amber);
+  display: inline-block;
+  vertical-align: middle;
+  animation: pulseFriction 2.8s ease-in-out infinite;
+}
+
+@keyframes pulseFriction {
```

Stylelint (17.11.1)

[error] Expected keyframe name "pulseFriction" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)